manifests/t/Twingate/Client/1.0.21.14067/Twingate.Client.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Twingate.Client
+PackageVersion: 1.0.21.14067
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://api.twingate.com/download/windows
+  InstallerSha256: F73772DC74E7D82506C7C748DBFCFB0FD92228430F2477802A557FD97E225E45
+ManifestType: installer

```

Comment:
```suggestion
  InstallerSwitches:
    Silent: /quiet
ManifestType: installer
```

Reply:
Validation is failing due to the install switches. This is what is being encountered: 
![image](https://user-images.githubusercontent.com/114952565/209867348-8636b55d-80df-479d-af11-800e78c22eba.png)

I've tested the above suggestions in a VM, and together they allow for a silent install.